src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+@import '../utils/mixins';
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), background properties should be combined into a shorthand property where possible. Consider using `background: center / contain no-repeat url(/src/images/star.svg);` to make the code cleaner.